documentation/cloud-manager-guide/admin/user-management/edit-user.md

```diff
@@ -0,0 +1,122 @@
+
+![image description](images/edit-deactivate-confirmation.png)
+
+The user's status will change from "Active" to "Inactive"
+
+![image description](images/edit-inactive-status.png)
+
+To update a user's status from the *Edit User* tab, adjust the *Status* toggle in the *User Details* section to the "Active" or "Inactive". Select *Save* to update the user's status.
+
+![image description](images/edit-user-inactive.png)
+
+After selecting save, you will return to the *Users* tab where the user status will update accordingly to the status you selected.
+
+![image description](images/edit-inactive-status.png)
+
+## Update role details
+
+To update the role or environment assignments of a user, select the **Options** (**...**) menu next to the role and environment pairing to be edited. Select **Edit** from the dropdown to enable editing for the role and environment fields.
+
+![image description](images/edit-roledetails.png)
+
+### Update user role
+
+To update the user role, select the down arrow located in the *Role* field to expand the dropdown menu.
+
+![image description](images/edit-role-arrow.png)
+
+From the list of roles, select the role to assign to the user.
+
+![image description](images/edit-select-role.png)
+
+Select the checkmark icon (check mark icon) to set the new role.
+
+![image description](images/edit-role-checkmark.png)
+
+Select **Save** to update the new user role.
+
+![image description](images/edit-save-role.png)
+
+### Update environment assignments
+
+To update environment assignments for a user, select the down arrow located in the *Environment* field to expand the dropdown menu.
+
+![image description](images/edit-env-arrow.png)
+
+To add new environments, select the environments from the dropdown menu. Multiple environments can be assigned to a user for a given role. After you have selected all of the required environment assignments, select the upward arrow in the *Environment* field to close the dropdown.
+
+![image description](images/edit-select-envs.png)
+
+To remove assigned environments, select the **X** located next to the environment name.
+
+![image description](images/edit-delete-envs.png)
+
+Once you have updated the user environment assignments, select the checkmark (checkmark icon) to set the new environment assignments.
+
+![image description](images/edit-env-checkmark.png)
+
+Select **Save** to update the user environment assignments.
+
+![image description](images/edit-save-envs.png)
+
+### Confirmation
+
+You can review the role and environment assignments for a user from the *Environment Access* tab. From the *Users* tab, select the name of the user to review.
+
+![image description](images/edit-select-username.png)
+
+The *Environment Access* tab of a user lists the environments they have been assigned to and their role associated with a specific environment. Review this list to ensure all role and environment assignments reflect your updates.
```

Comment:
Are we going to make guides to address the other tabs (VPN, DevSpace, Tokens)?

Reply:
Eventually, but not as part of mvp.

Reply:
RL confirmed these are being removed.